library/alloc/src/lib.rs

```diff
@@ -60,6 +60,7 @@
 // able to "empty" this crate. See <https://github.com/rust-lang/miri-test-libstd/issues/4>.
 // rustc itself never sets the feature, so this line has no affect there.
 #![cfg(any(not(feature = "miri-test-libstd"), test, doctest))]
+//
```

Comment:
This comment probably doesn't need to be here

Reply:
This was deliberate, to separate the miri-related attribute from what follows. Otherwise it seems that the Miri comment also applies to the later attributes.

Having an empty line without a comment doesn't work; rustfmt will remove that empty line.

Reply:
Hm, I don't think that's particularly clear. I think it's fine without, or adjusting the wording to make it clear that it only applies to the single attribute.

If you feel really strongly about it I don't care that much, though.

Reply:
I have reordered things a bit such that there are other comments that follow and naturally delimit the Miri attributes. The "Lints:" part existed in alloc but not std, so I also introduced it in std. Does that work for you?

Reply:
Yes, I saw that. It does work for me.